app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -28,6 +19,55 @@ def translator(action, _config)
           ]
         end
       end
+
+      # Docker orb specific implementation
+      class DockerOrb
+        def initialize
+          @config = {}
+        end
+
+        def matcher(orb, _conf)
+          orb.start_with?('docker/')
+        end
+
+        def translator(action, config)
+          @config = config
```

Comment:
if you are passing `config` through to the methods, the instance variable is not necessary